tasks/shared_libraries/recovery/shared_recovery.py

```diff
@@ -25,135 +24,96 @@ class OrcaStatus(Enum):
     Defines the status value used in the ORCA Recovery database for use by the recovery functions.
 
     """
-
     PENDING = 1
     STAGED = 2
-    SUCCESS = 3
-    FAILED = 4
+    FAILED = 3
+    SUCCESS = 4
 
 
-def post_status_for_job_to_queue(
+def create_status_for_job(
```

Comment:
Let Rizbi know of changes as this will affect his code. Same for the other functions modified in the shared library.

Reply:
The lambdas weren't updated to use the shared lib, so this shouldn't affect anything.